docs/reference/supported-environments.md

```diff
@@ -100,11 +100,11 @@ Requirements for the components can be seen below:
 
 | Component   | Java version | Other requirements                                                                                                                                                                                                |
 | ----------- | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Zeebe       | OpenJDK 21+  | Elasticsearch 8.13+<br/>Amazon OpenSearch 2.9+ (requires use of [OpenSearch exporter](../self-managed/zeebe-deployment/exporters/opensearch-exporter.md))                                                         |
-| Operate     | OpenJDK 21+  | Elasticsearch 8.13+<br/>Amazon OpenSearch 2.9+                                                                                                                                                                    |
-| Tasklist    | OpenJDK 21+  | Elasticsearch 8.13+<br/>Amazon OpenSearch 2.9+                                                                                                                                                                    |
+| Zeebe       | OpenJDK 21+  | Elasticsearch 8.16+<br/>Amazon OpenSearch 2.9+ (requires use of [OpenSearch exporter](../self-managed/zeebe-deployment/exporters/opensearch-exporter.md))                                                         |
+| Operate     | OpenJDK 21+  | Elasticsearch 8.16+<br/>Amazon OpenSearch 2.9+                                                                                                                                                                    |
+| Tasklist    | OpenJDK 21+  | Elasticsearch 8.16+<br/>Amazon OpenSearch 2.9+                                                                                                                                                                    |
 | Identity    | OpenJDK 17+  | Keycloak 25.x, 26.x<br/>PostgreSQL 14.x, 15.x or Amazon Aurora PostgreSQL 13.x, 14.x, 15.x (required for [certain features](/self-managed/identity/deployment/configuration-variables.md#database-configuration)) |
-| Optimize    | OpenJDK 21+  | Elasticsearch 8.13+<br/>Amazon OpenSearch 2.9+                                                                                                                                                                    |
+| Optimize    | OpenJDK 21+  | Elasticsearch 8.16+<br/>Amazon OpenSearch 2.9+                                                                                                                                                                    |
```

Comment:
@RomanJRW I updated Optimize here already, to reduce confusion (even if we haven't migrated the Optimize client yet). This is still part of https://github.com/camunda/product-hub/issues/2700,  As the clients are forward compatible this should work without issues. Let me know if you disagree with this.